perllib/FixMyStreet/App/Form/Waste/Bulky.pm

```diff
@@ -5,6 +5,8 @@ use DateTime::Format::Strptime;
 use HTML::FormHandler::Moose;
 extends 'FixMyStreet::App::Form::Waste';
 
+use constant MAX_ITEMS => 5;
```

Comment:
Please add an `XXX` comment to indicate that this needs to be un-hardcoded and fetched from config later on.